# HG changeset patch
# User jwe
# Date 857640449 0
#      Thu Mar 06 09:27:29 1997 +0000
# Node ID 32259460c627736b4abf4ca1f7637885bbf77d26
# Parent  ef422e6f613898f727571e522acb208ef667a5c1
[project @ 1997-03-06 09:27:29 by jwe]
info-emacs-octave-help

diff --git a/examples/info-emacs-octave-help b/examples/info-emacs-octave-help
--- a/examples/info-emacs-octave-help
+++ b/examples/info-emacs-octave-help
@@ -1,14 +1,14 @@
 #!/bin/sh
 # info-emacs-octave-help
 # Written by KH <Kurt.Hornik@ci.tuwien.ac.at> on 1996/07/01
-# Updated by KH on 1997/01/30
+# Updated by KH on 1997/03/04
 # Make Octave's `help -i' use Emacs octave-help.
 # Requires a running Emacs and gnuserv.
-cmd="(load \"octave-hlp\")"
+cmd="(require 'octave-hlp)"
 if [ $3 = "--directory" ];
 then
-	cmd="$cmd (add-to-list 'Info-directory-list \"$4\")"
-	shift 2
+    cmd="$cmd (add-to-list 'Info-directory-list \"$4\")"
+    shift 2
 fi
 cmd="$cmd (octave-help \"$4\")"
 gnudoit -q $cmd
